gcp/api/server.py

```diff
@@ -166,7 +166,24 @@ class OSVServicer(osv_service_v1_pb2_grpc.OSVServicer,
   def GetVulnById(self, request, context: grpc.ServicerContext):
     """Return a `Vulnerability` object for a given OSV ID."""
     bug = osv.Bug.get_by_id(request.id)
-    if not bug or bug.status == osv.BugStatus.UNPROCESSED:
+
+    if not bug:
+      # Check for aliases
+      alias_group = osv.AliasGroup.query(
+          osv.AliasGroup.bug_ids == request.id).get()
+      if alias_group:
+        string_builder = {
+            f'{alias} ' for alias in alias_group.bug_ids if alias != request.id
+        }
+        alias_string = ''.join(string_builder)
```

Comment:
nit: Is there a reason to use a set here (i.e. `{}` vs `[]`)
Also, instead of adding the space to the alias ID, you could do `' '.join(...)`
(and I guess it could also be one line)